web/components/TextBlock/VenuesSection/VenuesSection.tsx

```diff
@@ -20,26 +21,9 @@ export const VenuesSection = ({
     return null;
   }
 
-  switch (type) {
-    case 'all':
-      return (
-        <GridWrapper>
-          <VenueNames venues={allVenues} />
-        </GridWrapper>
-      );
-
-    case 'highlighted':
-      return (
-        <GridWrapper>
-          <VenueNames venues={venues} />
-        </GridWrapper>
-      );
-
-    case 'none':
-      return null;
-
-    default:
-      console.error(`Unrecognized VenuesSection type: '${type}'`);
-      return null;
-  }
+  return (
+    <GridWrapper>
+      <VenueNames venues={getCollectionForSelectionType(type, allVenues, venues)} />
+    </GridWrapper>
+  );
```

Comment:
This is simplified a tad too much. I would prefer that no empty `GridWrapper` elements are output to the DOM when there are no venues. So either keep the `none` special case from before, just like what Programs is doing right now even after the refactor, or (maybe even better) make this a conditional return based on the return value of `getCollectionForSelectionType` being non-empty.

E.g. something along the lines of (not tested)

```suggestion
  const selectedVenues = getCollectionForSelectionType(type, allVenues, venues);
  return selectedVenues.length > 0 && (
    <GridWrapper>
      <VenueNames venues={selectedVenues} />
    </GridWrapper>
  );
```

Reply:
Fixed in b129cc9.

Reply:
Thanks!